src/main/kotlin/tech/harmonysoft/oss/gradle/dist/CustomGradleDistributionPlugin.kt

```diff
@@ -9,16 +9,16 @@ class CustomGradleDistributionPlugin : Plugin<Project> {
     override fun apply(project: Project) {
         val config = project.extensions.create("gradleDist", CustomGradleDistConfig::class.java)
         project.afterEvaluate {
-            validateAndEnrich(config)
+            validateAndEnrich(config, project)
             project.tasks.register("buildGradleDist", BuildCustomGradleDistributionTask::class.java) {
                 it.config.set(config)
             }
         }
     }
 
-    private fun validateAndEnrich(config: CustomGradleDistConfig) {
+    private fun validateAndEnrich(config: CustomGradleDistConfig, project: Project) {
```

Comment:
we don't need to provide `Project` as a parameter here - it's already available as a property, other methods already use that approach